visualization-requirements.typ

```diff
@@ -29,7 +29,7 @@ The load balancer will be showcased at the innovation exposition in April. To he
     [#req("Must function as a useful debugging tool", mustHave)],
   ),
   caption: "General Visualization Requirements"
-)
+) <Requirements::visualization>
```

Comment:
This is going to cause the new lints in #282 to fail when this is merged. Lets do this for now.
```suggestion
) <Table::visualization-requirements>
```